tests/integration-tests/tests/update/test_update.py

```diff
@@ -412,6 +413,60 @@ def test_update_awsbatch(region, pcluster_config_reader, clusters_factory, test_
     _verify_initialization(region, cluster, updated_config)
 
 
+@pytest.mark.dimensions("eu-west-1", "c5.xlarge", "alinux2", "slurm")
+@pytest.mark.usefixtures("os", "instance")
+def test_update_compute_ami(region, pcluster_config_reader, clusters_factory, test_datadir):
+    # Create cluster with initial configuration
+    ec2 = boto3.client("ec2", region)
+    filters = [
+        {
+            "Name": "name",
+            "Values": ["aws-parallelcluster-" + get_installed_parallelcluster_version() + "-amzn2-hvm-x86_64*"],
+        }
+    ]
+    # Fixme choose private ami for develop branch vs public ami for release version
```

Comment:
it should be enough to have `Owners=["self", "amazon"]`

Reply:
done